src/replica/requests/SqlCreateTablesRequest.h

```diff
@@ -76,30 +80,31 @@ class SqlCreateTablesRequest : public SqlRequest {
      *   This variation of table schema will be used for the super-transaction-based
      *   ingest into the table.
      * @param columns Column definitions (name,type) of the table.
-     * @param onFinish (optional) A callback function to call upon completion of
+     * @param onFinish (optional) The callback function to call upon completion of
      *   the request.
-     * @param priority A priority level of the request.
-     * @param keepTracking Keep tracking the request before it finishes or fails.
-     * @param messenger An interface for communicating with workers.
+     * @param priority (optional) The priority level of the request.
+     * @param keepTracking (optional) Keep tracking the request before it finishes or fails.
+     * @param jobId (optional) an identifier of a job to which the request belongs
+     * @param requestExpirationIvalSec (optional) the time in seconds after which the request will expire.
+     *   The default value of '0' means an effective expiration time will be pull from the configuration.
```

Comment:
This is an excellent suggestion. Thank you! I may try it in this PR.

Reply:
I'm taking my words back ... unfortunately. After analyzing the design, I realized that passing a structure complicates the instantiating of these classes by adding more complexity. A problem is that subclasses of the base class `SqlRequest` also have many unique parameters.  So, I came up with a compromise - document all common parameters in the preamble for the base class:
```
/**
 * Abstract class SqlRequest is a base class for a family of the Controller-side
 * requests launching queries against a MySQL/MariaDB database of Qserv workers
 * via the corresponding Replication workers. The requests are sent over
 * the controller-worker protocol and they are executed by the the worker-side
 * framework.
 *
 * In case of a successful completion of a request an object of this request class
 * will receive a result set (if any) of the query.
 * 
 * The class is not intended to be used directly. Instead, it should be subclassed
 * by a specific request type. Objects of the subclassed request type are created
 * by a factory method "createAndStart" that is provided by each such class.
 * The following parameters of the method "createAndStart" are common for all subclasses:
 * @param controller the Controller associated with the request
 * @param workerName An identifier of a worker node.
 * @param onFinish (optional) The callback function to call upon completion of
 *   the request. The functin type is specific for each subclass.
 * @param priority (optional) The priority level of the request.
 * @param keepTracking (optional) Keep tracking the request before it finishes or fails.
 * @param jobId (optional) an identifier of a job to which the request belongs
 * @param requestExpirationIvalSec (optional) the time in seconds after which the request
 *   will expire. The default value of '0' means an effective expiration time will be pull
 *   from the configuration.
 */
class SqlRequest : public RequestMessenger {
```
Now I should only document the subclass-specific parameters where they are now. In this way the amount of duplicated comments should be eliminated. Here is an example for the class you posted your suggestion for:
```
     * @param database  The name of an existing database where the new table will be created.
     * @param table  The name of a table to be created.
     * @param engine  The name of the MySQL engine for the new table.
     * @param partitionByColumn  (optional, if not empty) The name of a column which
     *   will be used as a key to configure MySQL partitions for the new table.
     *   This variation of table schema will be used for the super-transaction-based
     *   ingest into the table.
     * @param columns  Column definitions (name,type) of the table.
     * @see the base class SqlRequest for the description of the common parameters of all subclasses.
```
Note that `@see` referencing to the base class.